From a35142a5ed5f32c021e7eaf6c33a5f5b8db61a1c Mon Sep 17 00:00:00 2001
From: "Enrico Weigelt, metux IT consult" <enrico.weigelt@gr13.net>
Date: Fri, 21 Oct 2016 17:55:39 +0200
Subject: [PATCH 8/9] stdlib: setenv.c: add brackets around ambigious if/else

---
 stdlib/setenv.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/stdlib/setenv.c b/stdlib/setenv.c
index e66045f..d8ad036 100644
--- a/stdlib/setenv.c
+++ b/stdlib/setenv.c
@@ -276,7 +276,8 @@ unsetenv (const char *name)
   LOCK;
 
   ep = __environ;
-  if (ep != NULL)
+  if ((ep != NULL))
+  {
     while (*ep != NULL)
       {
 	if (!strncmp (*ep, name, len) && (*ep)[len] == '=')
@@ -292,6 +293,7 @@ unsetenv (const char *name)
 	else
 	  ++ep;
       }
+  }
 
   UNLOCK;
 
-- 
2.6.4.442.g545299f

